packages/web/src/components/ControlledAutocomplete/index.jsx

```diff
@@ -132,6 +136,10 @@ ControlledAutocomplete.propTypes = {
   onBlur: PropTypes.func,
   onChange: PropTypes.func,
   options: PropTypes.array,
+  renderInput: PropTypes.func.isRequired,
+  showFormHelperText: PropTypes.bool,
+  showErrorTouched: PropTypes.bool,
```

Comment:
I can't seem to find where these two props are utilized. If they aren't, could we remove them?

Reply:
True, I left them by mistake. They got removed.